analyzers/rspec/cs/S4214_c#.html

```diff
@@ -29,4 +29,6 @@ <h2>Compliant Solution</h2>
     }
 }
 </pre>
+<h2>Deprecated</h2>
+<p>This rule is deprecated; use {rule:csharpsquid:S4200} instead.</p>
```

Comment:
@martin-strecker-sonarsource Should this go in already? 

It would be great to have a separate PR for this, to have a clear commit history and issue in the backlog to be propagated to release notes.

Reply:
Yes it should